### ISSUE TITLE
Glassfish upgrade: Edit Dataverse and edit Dataset pages doesn't save changes

### ISSUE THREAD

**Issue by scolapasta** [IQSS/dataverse#6448]
Broken out from #6230

Edit a dataverse, make changes, save - changes not saved to db. The same thing happens on edit user page.

**Comment by scolapasta** [comment]
Adding some debug lines, it appears the new values are not getting set into the dataverse object. So the save and merge do get called, but with all the old values.

**Comment by scolapasta** [comment]
I've updated the issue to reflect both pages where we saw this, because the problem and solution are the same for both.

Basically, the field are hidden behind a render that check the editmode. When you click edit, the edit mode gets changes and the field appears. The issue however is that editmode is a view param that gets passed with every request, so when you click save, edit mode gets changed back to null, rendered resoves to false, and the update_model_values phase ignores the fields.

While there are several possible ways to solve this (e.g. make the viewparam a different name and set the value of editmode in the init method), omnifaces provides a one character change fix. Using the omnifaces view param "doesn't call the converter and validators again on postbacks."

I don't think we need to change any other view params, but it possibly wouldn't hurt. We do need to be aware of when and how the values the view parameters point to are used for render logic.

For now, my plan is to be surgical and just change the editMode to o:view param.
